commands/cloudapi-v6/server.go

```diff
@@ -176,9 +192,9 @@ You can wait for the Request to be executed using ` + "`" + `--wait-for-request`
 	_ = create.Command.RegisterFlagCompletionFunc(cloudapiv6.ArgTemplateId, func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 		return completer.TemplatesIds(os.Stderr), cobra.ShellCompDirectiveNoFileComp
 	})
-	create.AddStringFlag(cloudapiv6.ArgType, "", serverEnterpriseType, "Type usages for the Server")
+	create.AddStringFlag(cloudapiv6.ArgType, "", serverEnterpriseType, "Type usages for the Server. Can be one of: ENTERPRISE, CUBE or VCPU")
```

Comment:
You can (but aren't forced to) use our custom `AddSetFlag` to limit to certain values and automatically add completions and "Can be one of..." help text for those values